packages/orbit-components/src/Separator/types.d.ts

```diff
@@ -83,9 +83,19 @@ export type PaletteTokens =
   | "paletteSocialFacebookHover"
   | "paletteSocialFacebookActive";
 
-export interface Props extends Common.SpaceAfter {
+export interface CommonProps extends Common.SpaceAfter {
   indent?: Indent;
-  type?: "solid" | "dashed" | "dotted" | "double" | "none";
-  color?: PaletteTokens;
   align?: Align;
 }
+
+type ConditionalSeparatorType =
+  | {
+      readonly color: PaletteTokens;
```

Comment:
I think we want this
```suggestion
      readonly color?: PaletteTokens;
```

In the past, we were able to have a type without defining a color. If we don't make the `color` optional here, there will be an error requiring `color` to be defined.